src/main/java/uk/gov/pay/ledger/transaction/search/common/TransactionSearchParamsValidator.java

```diff
@@ -1,18 +1,26 @@
 package uk.gov.pay.ledger.transaction.search.common;
 
 import uk.gov.pay.ledger.exception.UnparsableDateException;
+import uk.gov.pay.ledger.exception.ValidationException;
 
 import java.time.ZonedDateTime;
 import java.time.format.DateTimeParseException;
+import java.util.Collections;
 
+import static java.lang.String.format;
+import static org.apache.commons.lang3.StringUtils.isBlank;
 import static org.apache.commons.lang3.StringUtils.isNotBlank;
 
 public class TransactionSearchParamsValidator {
 
+    private static final String GATEWAY_ACCOUNT_ID = "account_id";
     private static final String FROM_DATE_FIELD = "from_date";
     private static final String TO_DATE_FIELD = "to_date";
 
     public static void validateSearchParams(TransactionSearchParams searchParams) {
+
+        throwIfBlankFieldValue(GATEWAY_ACCOUNT_ID, searchParams.getAccountId());
```

Comment:
Do we want to always enforce this value?

Reply:
Yes, it is always required for now